### PR TITLE
Update readme version advice to be generic

### PR DESCRIPTION
**Pull request by n-hebert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ on: push
 jobs:
   compile-artifact:
     name: "Compile artifact zip"
-    uses: rewindio/github-action-eb-deploy/.github/workflows/compile-artifact.yml@v2.0.0
+    # Please replace all X.Y.Z values with the latest tag
+    uses: rewindio/github-action-eb-deploy/.github/workflows/compile-artifact.yml@vX.Y.Z
```
Comment (davegallant) [inline README.md:53]: Might be nice to support a `vX` like actions/checkout:
https://github.com/actions/checkout/tags to get small fixes for free.

**Review by davegallant — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (n-hebert) [reply README.md:53]: Hm! That's a good point, but we'd need to force push the tag.
Would that pose any issue?

Comment (davegallant) [reply README.md:53]: We're not signing tags with GPG or preventing tags to be overridden ATM.

Comment (n-hebert) [reply README.md:53]: I'll look to add some automation on this, as we'll want to make sure the tags stay updated properly, and for now stick with what we got to not move too quickly.

**Pull request merged**